docs/learn-more/faq.md

```diff
@@ -127,3 +130,15 @@ IOError: [Errno 13] Permission denied: u'./.env'
 ```
 
 This may happen if you have installed Docker from a snap package. This installation mode is not supported, please install it following [the official documentation](./quickstart/1-install-requirements.md#install-docker) (See [#78](https://github.com/src-d/sourced-ce/issues/78)).
+
+
+## Why Do I Need Internet Connection?
+
+source{d} CE automatically fetches some resources from the Internet when they are not found locally:
+
+- the source{d} CE configuration is fetched automatically when initializing it for the first time, using the proper version for the current version of `sourced`, e.g. if using `v0.16.0` it will be automatically fetched `https://raw.githubusercontent.com/src-d/sourced-ce/v0.16.0/docker-compose.yml`.
```

Comment:
```suggestion
- the source{d} CE configuration is fetched automatically when initializing it for the first time, using the proper version for the current version of `sourced`, e.g. if using `v0.16.0` it will automatically fetch `https://raw.githubusercontent.com/src-d/sourced-ce/v0.16.0/docker-compose.yml`.
```